### PR TITLE
Rebrand blog layout

### PR DESCRIPTION
**Pull request by kapowaz**
This PR encapsulates an entirely new design for the engineering blog, with pages for guidelines and authors, plus styles for blog posts and guideline documents.

### CODE REVIEW

**Comment by tommyp** [comment]
Screenshot?

**Comment by jphastings** [comment]
Looks really good to me - screenshot would be nice though :)

**Comment by kapowaz** [comment]
I’m just creating some screenshots at different breakpoints — will post them here shortly!

**Comment by kapowaz** [comment]
These views are fluidly-responsive, so will adapt to widths between these breakpoints, but for illustration purposes I’ve taken screenshots at 1280px, 640px and 375px (it should work all the way down to 320px though). Naturally I needed some dummy data to test layouts, which isn’t actually part of this PR — there’s only a single blog post currently (and a single author, @mezis).

**Homepage**

1280px viewport

homepage-1280

640px viewport

homepage-640

375px viewport

homepage-375

**Navigation**

This is essentially the same system as is currently live.

640px viewport

navigation-640

375px viewport

navigation-375

**Articles**

1280px viewport

articles-1280

640px viewport

articles-640

375px viewport

articles-375

**Guidelines**

1280px viewport

guidlines-1280

640px viewport

guidelines-640

375px viewport

guidelines-375

**Authors**

1280px viewport

authors-1280

640px viewport

authors-640

375px viewport

articles-375

File: _authors/julien-letessier.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
```
Comment (mischizzle) [inline _authors/julien-letessier.md]: Where's your schnazzy bio Ben? ;)

**Comment by mischizzle** [comment]
Looks good - thanks for updating!

File: _authors/julien-letessier.md (hunk 1 above)
Comment (kapowaz) [inline _authors/julien-letessier.md]: I have no blog posts yet, but I will write one once I do!

**Comment by tommyp** [comment]
@kapowaz For future reference Markdown tables are good for screenshots. https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables

**Pull request merged**